recipes/pyaqsapi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyaqsapi" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.9" %}
+{% set setuptools_min = "77.0.3" %}
```

Comment:
```suggestion
```

Also you don't have to explicitly set `python_min` to 3.9 here since that's conda-forge's default global pin